File: 1101 - Sequencia de Numeros e Soma/sequencia_de_numeros_e_soma.py
import sys

for linha in sys.stdin:
    partes = linha.split()
    if len(partes) < 2:
        continue
    m, n = int(partes[0]), int(partes[1])
    if m <= 0 or n <= 0:
        break
    if m > n:
        m, n = n, m
    soma = 0
    for i in range(m, n + 1):
        print(i, end=" ")
        soma += i
    print(f"Sum={soma}")

File: 1101 - Sequencia de Numeros e Soma/sequencia_de_numeros_e_soma.c
#include <stdio.h>

int main(){
    int m, n;
    while(scanf("%d %d", &m, &n) == 2){
        if(m <= 0 || n <= 0) break;
        if(m > n){ int t = m; m = n; n = t; }
        int soma = 0;
        for(int i = m; i <= n; i++){
            printf("%d ", i);
            soma += i;
        }
        printf("Sum=%d\n", soma);
    }
    return 0;
}
